### PR TITLE
Added Input Component

### PR DESCRIPTION
**Pull request by nareshpoonia**
Added Input Component

### CODE REVIEW

File: html/docs.html [hunk 1]
```diff
@@ -422,14 +422,68 @@ <h2 class="medium-heading docs-heading">Round Image</h2>
...
+                <h2 class="medium-heading docs-heading" id="input">
+                  Input Textbox
+                </h2>
+                <div class="sample-container m-2">
+                  <figure>
+                    <input class="input-textbox simple my-8p outline-none p-8p flex-center" type="text" placeholder="Type your query here..."/>
+                  </figure>
+                </div>
+                <div>
+                  <figure class=" m-2 code-container">
```
Comment (saadahmadhere) [inline html/docs.html]: remove space before m-2

File: styles/utility.css [hunk 2]
```diff
@@ -92,13 +114,21 @@ code {
...
+.border-primary {
+  border-color: var(--primary-color);
+}
+.border-solid {
+  border-style: solid;
+}
+
+.border-green-600 {
+  border-color: rgb(22 163 74);
+}
```
Comment (saadahmadhere) [inline styles/utility.css:134]: add EOF line here.

**Review by saadahmadhere — APPROVED** [review]
Well written. Clean and sleek. Good use of variables.

**Review by RakshithBhat03 — APPROVED** [review]
Everything looks good. Good use of variables and the code looks well structured. Please make changes suggested by the other reviewer.

File: html/docs.html (hunk 1 above)
Comment (nareshpoonia) [reply html/docs.html]: Changes made, thanks for the input

File: styles/utility.css (hunk 2 above)
Comment (nareshpoonia) [reply styles/utility.css:134]: EOF is showing in VS Code, but somehow not showing here.
Tried adding it but prettier is removing that.

**Pull request merged**